.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+**Point to the API documentation/Plugin/feature? Please describe.**
+
+A clear and concise description of the plugin you have added or the entity you have updated
+
+**Presentation that this code works**
+
+An Example that the code works. You can exercise the code as raw API calls or using any other method
+
+`Eg: # Read Snapshot
```

Comment:
I think you want fenced code block, like this:

~~~
```
Eg: # Read Snapshot
    Snapshot(host=<host_id>, id=<snapshot_id>).read()
```
~~~

With only single `` ` ``, this code will be all in one line